packages/ui/src/components/Table/TablePagination.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, TableCell, TableFooter, TableRow } from '@mui/material';
+import usePagination from '@mui/material/usePagination/usePagination';
+import { FC } from 'react';
+
+import { TablePaginationItem } from './TablePaginationItem';
+import { PageChangeHandler } from './types';
+
+interface TablePaginationProps {
+    itemsCount: number;
+    currentPage: number;
+    rowsPerPage: number;
+    onChangePage?: PageChangeHandler;
+}
+
+export const TablePagination: FC<TablePaginationProps> = ({ itemsCount, currentPage, rowsPerPage, onChangePage }) => {
```

Comment:
can we have a small test for this that checks if the proper pages are displayed and changed based on what's clicked?